marimo/_messaging/cell_output.py

```diff
@@ -37,16 +39,30 @@ class CellOutput:
     def asdict(self) -> dict[str, Any]:
         return asdict(self)
 
+    def __post_init__(self) -> None:
+        # For plain text outputs, we sanitize the message
+        # if it's already code highlighted.
+        if (
+            self.mimetype == "text/plain"
+            and isinstance(self.data, str)
+            and not is_code_highlighting(self.data)
```

Comment:
should this be `and is_code_highlighting`?

Reply:
@mscolnick this feels brittle and adds a linear-time string search to every text output.

Instead, I think we may just want to send the right mimetype when writing a traceback. I can try to make a patch.